decent-dao/fractal-interface
# Show token prices

## Description

This PR implements fetching coin prices from CoinGecko.
The implementation is as dumb as you could only imagine. Probably we could save some requests to putting returned value of `useFormatCoins` into global context - but that might lead us into unnecessary implementation over-engineering (given the fact that we could be migrating to the Safe API Kit once they'll release pricing service anyway).


## Notes



## Issue
Closes #1362 


## Testing
1. Visit some mainnet DAO with bunch of assets
2. Verify that prices for token and total USD is calculated properly


## Screenshots (if applicable)
